Table/Base.php

```diff
@@ -434,6 +434,28 @@ public function setPaginator(Paginator\BaseSource $paginator)
         $this->_paginator = $paginator;
     }
 
+    /**
+     * Return the table's Zend paginator.
+     *
+     * @return string
+     */
+    public function getZendPaginator()
+    {
+        return $this->_zendPaginator;
+    }
+
+    /**
+     * Set the table's Zend paginator.
+     *
+     * @param \Zend\Paginator\Paginator $paginator Zend paginator instance.
```

Comment:
Does ZF1 have a compatible equiv. to this class? Or were there breaking changes along with the move to ZF2's namespaced classes? If ZF1 has a compatible class just in a different namespace it would be good to support it as ZTal generally supports ZF1. I guess this would mean removing the hype hint on the setter and using in-function checks for instanceof

Reply:
The Zend\Paginator package is significantly different from the old Paginator component of ZF1, though I haven't specifically tested for backwards-compatibility. Given this is new functionality, and completely optional, I don't see a particular need to maintain support for the old ZF1 version. Also, it could cause problems when using our own Dada\Paginator, since it extends Zend\Paginator.

Reply:
Successfully tested against paginator included with Zend Framework 1